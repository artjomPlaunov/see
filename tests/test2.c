void func () {
  int a = 5;
  int var = 7;
}

int main () {
  
}
