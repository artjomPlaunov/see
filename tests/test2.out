Function Name: func
Return Type: VOID
Declaration List of type INT:
  (ID: a) = (VALUE: 5)
Declaration List of type INT:
  (ID: var) = (VALUE: 7)

Function Name: main
Return Type: INT

